src/App.tsx

```diff
@@ -1,61 +1,34 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { getUserById } from './utils/getUserById';
+
+export const App: React.FC = () => {
+  const preparedTodos: Todo[] = todosFromServer.map(todo => {
+    return {
+      ...todo,
+      user: getUserById(todo.userId) || null,
+    };
```

Comment:
[CODE KNOWLEDGE] - You don't need to create a copy of the `todosFromServer` array when using map as it does not mutate the original array. You can directly call `todosFromServer.map` without any issues.